skrifa/src/outline/autohint/axis.rs

```diff
@@ -184,6 +184,10 @@ impl Segment {
         edges.get(self.edge_ix.map(|ix| ix as usize)?)
     }
 
+    pub fn edge_next<'a>(&self, segments: &'a [Segment]) -> Option<&'a Segment> {
```

Comment:
is there a reason this isn't `next_edge`? Does it mean something different?

Reply:
This returns the next segment in the parent edge. This name was copied from FT but I agree that it's ambiguous. Renamed to `next_in_edge` for clarity.